client/my-sites/marketing/main.jsx

```diff
@@ -123,6 +126,22 @@ export const Sharing = ( {
 		titleHeader = translate( 'Integrations' );
 	}
 
+	// Determine the right type of upsell according to the plan.
+	// For the Free plan and the Starter plan, we want to upsell the no-ads add-on.
+	// Otherwise, it upsells the Pro plan.
+	let upsellNudgeTitle = translate( 'No ads with WordPress.com Pro' );
+	let upsellNudgeDesc = translate( 'Prevent ads from showing on your site.' );
+	let upsellNudgeHref;
+
+	if (
+		isStarterPlanEnabled() &&
+		( isFreePlan( currentPlanSlug ) || isStarterPlan( currentPlanSlug ) )
```

Comment:
I don't see why we need any of the logic above?... I think it should be possible to just replace the existing Pro plan text with the No Ads text instead.

Once this is launched, we want to upsell the No Ads add-on to everyone who doesn't have the No Ads feature; as far as I understand, there is no scenario we'd want to upsell the Pro plan going forward, right?  And I think the `UpsellNudge` component itself already takes care of the logic of only showing this upsell when it's actually relevant (i.e., when the site doesn't have the feature)?

Reply:
I see your point; we shouldn't be releasing this until the No Ads addon is implemented.
I removed the check and needless description.

Thanks for pointing these things out!